### ISSUE TITLE
Complain about lack of sufficient open gl[es] versions

### ISSUE THREAD

**Issue by fitzgen** [servo/webrender#1372]
I've been getting "failed to create gl context" errors which get unwrapped as panics, and after working with folks on irc, I've come to realize this is because I don't have a high enough version of open gl(es?). It would be great if this were explicitly stated and I didn't have to hunt this information down (especially when this only happens under rr, for example because it forces software rendering).

**Comment by jrmuizel** [comment]
FWIW, it is possible to use webrender with software rendering under rr. llvmpipe should work fine however, I know that it doesn't work on Fedora because they disable texture float because of patent concerns.

**Comment by fitzgen** [comment]
Yes, this is the issue I've been running into (among others). It would be great if I was just informed of the missing requirements up front instead of having to track them down and investigate myself.

Currently building mesa from source to enable texture floats.
